external/esp_idf_port/esp32/esp32_wifi_os_adapter.c

```diff
@@ -424,7 +424,7 @@ static void IRAM_ATTR timer_setfn_wrapper(void *ptimer, void *pfunction, void *p
 	ETSTimer *etimer = (ETSTimer *) ptimer;
 
 	if (etimer->timer_period != TIMER_INITIALIZED_VAL) {
-		memset(ptimer, 0, sizeof(*ptimer));
+		memset(etimer, 0, sizeof(*etimer));
```

Comment:
@zhouxinhe 
i think, sizeof(ETSTimer) is better.

Reply:
fixed, thanks